src/plugins/explore/public/application/index.tsx

```diff
@@ -25,16 +26,22 @@ interface ExploreRouteProps {
 }
 
 type ExploreComponentProps = ExploreRouteProps &
-  Partial<Pick<AppMountParameters, 'setHeaderActionMenu'>>;
+  Partial<Pick<AppMountParameters, 'setHeaderActionMenu'>> & { dataSetup: DataPublicPluginSetup };
 
 const renderExploreFlavor = (flavor: ExploreFlavor, props: ExploreComponentProps) => {
   switch (flavor) {
     case ExploreFlavor.Logs:
-      return <LogsPage setHeaderActionMenu={props.setHeaderActionMenu} />;
+      return (
+        <LogsPage setHeaderActionMenu={props.setHeaderActionMenu} dataSetup={props.dataSetup} />
```

Comment:
eventually if any access to services and plugins. it would make more sense to use the OpenSearch Dashboards react context

Reply:
@FriedhelmWS I think you can use `data` from the context which is `DataPublicPluginStart`

Reply:
Done, sorry for didn't noticed that QueryStart and SearchStart are also valid to be used.